repositories/vaccineOps.js

```diff
@@ -0,0 +1,20 @@
+const axios = require("axios");
+
+module.exports = {
+  getUsData: async (state) => {
+    //Get current data for most recent date
+    var yesterday = new Date(new Date().valueOf() - 1000 * 60 * 60 * 24)
+      .toISOString()
+      .slice(0, 10);
+    let apicall =
+      "https://data.cdc.gov/resource/unsk-b7fc.json?location=" +
+      state +
+      "&date=" +
+      yesterday +
+      "T00:00:00.000";
```

Comment:
nitpick: you can use backtics (`) to create format strings in JavaScript

```suggestion
const url = `https://data.cdc.gov/resource/unsk-b7fc.json?location=${state}&date=${yesterday}T00:00:00.000`;
```

also changed the variable name and made it a `const` since it isn't being reassigned.